patches/minecraft/net/minecraft/world/WorldProviderEnd.java.patch

```diff
@@ -0,0 +1,17 @@
+--- ../src-base/minecraft/net/minecraft/world/WorldProviderEnd.java
++++ ../src-work/minecraft/net/minecraft/world/WorldProviderEnd.java
+@@ -16,6 +16,14 @@
+ {
+     private DragonFightManager field_186064_g = null;
+ 
++    /*======================================= Forge Start =========================================*/
++    public WorldProviderEnd()
```

Comment:
Unless you have an alternative to pushing this in here - do it in the constructor of `WorldProvider`?